hudi-common/src/main/java/org/apache/hudi/common/table/view/HoodieTableFileSystemView.java

```diff
@@ -307,6 +307,10 @@ void removeFileGroupsInPendingClustering(Stream<Pair<HoodieFileGroupId, HoodieIn
    */
   @Override
   Stream<HoodieFileGroup> fetchAllStoredFileGroups(String partition) {
+    if (!isPartitionAvailableInStore(partition)) {
```

Comment:
We can `partitionToFileGroupsMap.get(partition)` first then decide if it is now to eliminate one lookup for the cache.

Reply:
Had changed，do you mean modify like current way？ @danny0405